enable-wss-diagnostics/arm template and bicep/README.md

```diff
@@ -0,0 +1,18 @@
+# AzSHCI-Automation
+
+Manage Diagnostic Level and Windows Server Subscription settings for Azure Stack HCI clusters using PowerShell and Azure CLI.
+
+Pre-requisites:
+	1. Create Workspace 'microsoft.operationalinsights/workspaces' manually before running script.
```

Comment:
why is this needed

Reply:
There was a similar Readme file for ama installation, so to have consistency I added these. If they are unnecessary, I can remove them.